### PR TITLE
added Modal Component

### PR DESCRIPTION
**Pull request by mansi2011**
Added Modal Component in my component library

### CODE REVIEW

**Review by Prabhav-Sharma — APPROVED** [review]
Code is clean. 
Suggestion: Add a preview in the description and use any extension in VS code for reformatting code, ex. Prettier or beautify.

File: Components/Modal/Modal.css [hunk 1]
```diff
@@ -0,0 +1,89 @@
+.dialogs{
```
Comment (tanishk15sharma) [inline Components/Modal/Modal.css:1]: I liked how you are centering the dialog box

File: Components/Modal/Modal.css [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+
+
+
+
+.dialog-btn{
+  
+    border-style: none;
+    color: var(--dialog-btn-color);
+    font-size: large;
+    font-weight: bold;
```
Comment (tanishk15sharma) [inline Components/Modal/Modal.css:41]: Suggestion , You can give here some value like `500` , `600` ,etc

File: Components/Modal/Modal.html [hunk 3]
```diff
@@ -0,0 +1,147 @@
...
+                    style="width: 664px; height: 200px; border:0; transform: scale(1); overflow:hidden; "
+                    sandbox="allow-scripts allow-same-origin">
+                    </iframe>
+                </div>
+
+            </div>
+            
+<!-- 
+            <div class="carbon-container">
```
Comment (tanishk15sharma) [inline Components/Modal/Modal.html]: you can remove this commented code .

**Review by tanishk15sharma — APPROVED** [review]
I liked the UI and how you are using css properties like for centering `div` and variable classes for colors . You can remove commented `iframe` tags as they are of no use . you can merge this `Branch`  .

File: Components/Modal/Modal.css (hunk 2 above)
Comment (mansi2011) [reply Components/Modal/Modal.css:41]: okay will try using it

File: Components/Modal/Modal.html (hunk 3 above)
Comment (mansi2011) [reply Components/Modal/Modal.html]: done!

**Pull request merged**